source/deployment/services/infrastructure.rst

```diff
@@ -21,6 +21,8 @@ The common role includes the following services:
 HAProxy
 =======
 
+Please read certificate configuration (https://docs.osism.io/configuration/environments/openstack.html#haproxy)
+
```

Comment:
Please use cross references (https://www.sphinx-doc.org/en/1.5/markup/inline.html).

Reply:
I tried these, I hope this works.